src/index.html

```diff
@@ -3,11 +3,347 @@
+            alt="product"
+            class="creators__img"
+          >
+        </div>
+      </section>
+
+      <section class="ayurvedic">
+        <h4 class="h4 ayurvedic__title">Ayurvedic Tooth Powder</h4>
+
+        <div class="ayurvedic__consist">
+          <div class="ayurvedic__consist-items">
+            <p class="p1 ayurvedic__consist-item">Kaolin</p>
+            <p class="p1 ayurvedic__consist-item">Sivak</p>
+            <p class="p1 ayurvedic__consist-item">Calcium</p>
+            <p class="p1 ayurvedic__consist-item">Licorice root</p>
+            <p class="p1 ayurvedic__consist-item">Turmeric</p>
+          </div>
+
+          <div class="ayurvedic__img"></div>
+        </div>
+
+        <a href="./prodcuct-page.html" class="ayurvedic__btn">LEARN MORE</a>
+      </section>
+
+      <section class="benefits">
+        <article class="benefits__item">
+          <img
+            src="./images/icons/fabric.png"
+            alt="benefit"
+            class="benefits__img"
+          >
+          <p class="benefits__title p2">No synthetics and microplastics</p>
+        </article>
+
+        <article class="benefits__item">
+          <img
+            src="./images/icons/recycle.png"
+            alt="benefit"
+            class="benefits__img"
+          >
+          <p class="benefits__title p2">product packaging can be recycled or reused</p>
+        </article>
+
+        <article class="benefits__item">
+          <img
+            src="./images/icons/leaf.png"
+            alt="benefit"
+            class="benefits__img"
+          >
+          <p class="benefits__title p2">only natural ingredients</p>
+        </article>
+      </section>
+
+      <section class="contact" id="contact">
+        <div class="contact__description">
+          <h4 class="h4 contact__title">contact us</h4>
+          <p class="p1 contact__info">We will help you choose a product that suits you and answer your other questions</p>
+
+          <form
+            action="#"
+            class="contact__form form"
+            method="get"
+            id="form"
+          >
+            <input
+              type="text"
+              placeholder="Your Name"
+              name="name"
+              class="form__name form__input"
+              required
+
+            >
+            <input
+              type="email"
+              placeholder="E-Mail"
+              name="email"
+              class="form__email form__input"
+              required
+            >
+            <input
+              type="tel"
+              placeholder="Phone Number"
+              name="phone"
+              class="form__phone form__input"
+              required
+            >
+            <input
+              type="text"
+              placeholder="Your Message"
+              name="message"
+              class="form__message form__input"
+              required
+            >
+            <input
+            type="reset"
```

Comment:
use [reset method](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset)

```suggestion
            type="submit"
```